teku/src/main/java/tech/pegasys/teku/cli/options/Eth2NetworkOptions.java

```diff
@@ -269,6 +269,19 @@ public class Eth2NetworkOptions {
       arity = "0..1")
   private String epochsStoreBlobs;
 
+  @CommandLine.Option(
+      names = {"--Xincident-intervals"},
+      hidden = true,
+      paramLabel = "<STRING>",
+      description =
+          "Sets the interval where network is considered to be in incident. "
+              + "All the slots in these epochs should be empty. Any chain with blocks "
+              + "inside interval is considered invalid.",
```

Comment:
hrm so thats interesting, im not sure any epochs were completely empty...

Reply:
it's expected that it must fork so all the slots inside incident intervals are completely empty